_preload.lua

```diff
@@ -224,6 +224,12 @@
 		tokens = true,
 	}
 	
+	api.register {
+		name = "androidapplibname",
+		scope = "project",
```

Comment:
I am unsure how android deals with the packages, but if we want to have different lib names for different configs, the scope for this property should be "config", not "project", since the other change refers to cfg.androidapplibname